porcupine/default_keybindings.tcl

```diff
@@ -60,6 +60,9 @@ event add "<<Menubar:Edit/Find and Replace>>" <$control_ish-f>
 # fold plugin
 event add "<<Menubar:Edit/Fold>>" <$alt_ish-f>
 
+# comment/uncomment plugin
+event add "<<Menubar:Edit/Comment//uncomment selected lines>>" <$control_ish-7>
```

Comment:
For people with american keyboards, it would be better if they could use the slash character:

```suggestion
event add "<<Menubar:Edit/Comment//uncomment selected lines>>" <Shift-slash>
```

On my system this works the same as `<$control_ish-7>`. Does it work for you?

Reply:
`<Shift-slash>` does not work on my system. Can't find any button combination that triggers this.

Reply:
Sorry, my bad. I keep forgetting that this plugin doesn't support commenting out a single line without explicitly selecting text. `Shift-slash`does work for me.